concordium-contracts-common/src/types.rs

```diff
@@ -1250,6 +1351,21 @@ impl fmt::Display for NewAttributeValueError {
 #[derive(Clone, Copy, Debug, Ord, PartialOrd, Eq, PartialEq)]
 pub struct AttributeTag(pub u8);
 
+#[cfg(feature = "concordium-quickcheck")]
+impl quickcheck::Arbitrary for AttributeTag {
+    // We generate only valid tags, that is indices into the `ATTRIBUTE_NAMES` array
+    // defined in `concordium-base/rust-src/id/src/types.rs`
```

Comment:
Why is that the choice that is made?

We discussed that it might be sensible to generate arbitrary tags to make the contract more robust with respect to future additions.

Reply:
Yes, indeed. I relaxed the constraint, so a tag is just a random `u8` now.